clang/test/3C/typedefs.c

```diff
@@ -44,6 +46,14 @@ int foo(void) {
     bad b = (int*) 3;
     //CHECK: bad b = (int*) 3;
     badP b2 = (intptr*) 3;
+    typedef int nat;
+    const nat z = 3;
+    const nat* cnstp = &z;
+    //CHECK: _Ptr<const nat> cnstp = &z;
+    int w = 34;
+    const integer c = &w;
+    //CHECK: const integer c = &w;
+
```

Comment:
Can you add a test case where there is a `const` qualifier inside the typedef definition? I think that it should just work with the code as written.